api/envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.proto

```diff
@@ -27,10 +27,19 @@ message GcpAuthnFilterConfig {
   // Retry policy for fetching tokens.
   // This field is optional. If it is not configured, the filter will be fail-closed (i.e., reject the requests).
   config.core.v3.RetryPolicy retry_policy = 2;
+
+  // Token cache configuration. This field is optional.
+  TokenCacheConfig cache_config = 3;
 }
 
 // Audience is the URL of the receiving service that performs token authentication.
 // It will be provided to the filter through cluster's typed_filter_metadata.
 message Audience {
   string url = 1 [(validate.rules).string = {min_len: 1}];
 }
+
+// Token Cache configuration.
+message TokenCacheConfig {
+  // Cache size. Default value is 0, which means the cache is disabled by default.
+  uint32 cache_size = 1;
```

Comment:
What are the units here?  Bytes?  Number of cache entries?  Please document in comments.

Is `uint32` the right size here?  Could we possibly need `uint64` in the future?  If so, we should use that type now, because we won't be able to change it later.

As described in https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md, this should probably use one of the wrapper scalar types, so that the control plane can differentiate between "unset" and "0".  That allows data plane implementations to choose their own defaults if the control plane does not care.